packages/react/src/variables.scss

```diff
@@ -27,6 +30,15 @@ $texteditor-edit-color: $texteditor-color;
 $list-padding-left: $material-list-item-horizontal-padding;
 $footer-height: 70px;
 $footer-left-padding: math.div(($side-panel-min-width - $material-base-icon-size), 2);
+$tooltip-danger-button-color: button_colors.$button-danger-text-hover-bg;
+$tooltip-success-button-color: button_colors.$button-success-text-hover-bg;
+
+$theme-padding: $material-button-horizontal-padding;
+// WA for no button-height in generic theme, remove when design-system is released
+$button-height: calc(26px + ($theme-padding - 10px)/3);
```

Comment:
The calculation is strange& Could we get rid of it?

Reply:
We have button height scss variable in material theme, but doesn't have it in generic
Pavel insisted on calculation of toolbar header height relative to button height
And there are also some places in code where we use it
I agree, that this calculations look kinda strange, but i can come up with other solution, only dropping this customization